src/client/game.cpp

```diff
@@ -2225,12 +2226,13 @@ void Game::toggleMinimap(bool shift_pressed)
 			m_game_ui->showTranslatedStatusText("Minimap in radar mode, Zoom x4");
 			break;
 		default:
-			mode = MINIMAP_MODE_OFF;
 			m_game_ui->m_flags.show_minimap = false;
-			if (hud_flags & HUD_FLAG_MINIMAP_VISIBLE)
+			if (!mapper->isDisabled()) {
```

Comment:
Turning off should always be possible, also when the minimap is disabled (but shown for some reason).

Reply:
Good idea.